doc/python/Dirac.py

```diff
@@ -1,26 +1,29 @@
-
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
 import sys
-from sympy import symbols,sin,cos
-from printer import Format,xpdf,Get_Program,Print_Function
-from ga import Ga
+from sympy import symbols, sin, cos
+from galgebra.printer import Format, xpdf, Get_Program, Print_Function
+from galgebra.ga import Ga
 
 Format()
-coords = symbols('t x y z',real=True)
-(st4d,g0,g1,g2,g3) = Ga.build('gamma*t|x|y|z',g=[1,-1,-1,-1],coords=coords)
+coords = symbols('t x y z', real=True)
+(st4d, g0, g1, g2, g3) = Ga.build(
+    'gamma*t|x|y|z', g=[1, -1, -1, -1], coords=coords)
 I = st4d.i
 
-(m,e) = symbols('m e')
+(m, e) = symbols('m e')
 
-psi = st4d.mv('psi','spinor',f=True)
-A = st4d.mv('A','vector',f=True)
-sig_z = g3*g0
+psi = st4d.mv('psi', 'spinor', f=True)
+A = st4d.mv('A', 'vector', f=True)
+sig_z = g3 * g0
 
-print '\\text{4-Vector Potential\\;\\;}\\bm{A} =',A
-print '\\text{8-component real spinor\\;\\;}\\bm{\\psi} =',psi
+print('\\text{4-Vector Potential\\;\\;}\\bm{A} =', A)
+print('\\text{8-component real spinor\\;\\;}\\bm{\\psi} =', psi)
 
-dirac_eq = (st4d.grad*psi)*I*sig_z-e*A*psi-m*psi*g0
+dirac_eq = (st4d.grad * psi) * I * sig_z - e * A * psi - m * psi * g0
 dirac_eq = dirac_eq.simplify()
 
-dirac_eq.Fmt(3,r'%\text{Dirac Equation\;\;}\nabla \bm{\psi}'+\
-               r' I \sigma_{z}-e\bm{A}\bm{\psi}-m\bm{\psi}\gamma_{t} = 0')
-xpdf(paper='landscape',prog=True)
+# FIXME terms of \psi^{ty} are not grouped together
```

Comment:
All codes in doc/python are somehow not working in a non-trivial-to-fix way, but most of the prints are correct. All incorrect places are compared with my eyes and marked with FIXME. Still working on all the fixes.